src/riak_kv_app.erl

```diff
@@ -207,6 +207,12 @@ start(_Type, _StartArgs) ->
                                           [enabled, disabled],
                                           disabled),
 
+            riak_core_capability:register({riak_kv, riak_ql_ddl_version},
+                                           riak_ql_ddl:get_compiler_capabilities(),
+                                           lists:last(riak_ql_ddl:get_compiler_capabilities())),
+
```

Comment:
Agree that this registration is probably premature, so we really should just reschedule https://bashoeng.atlassian.net/browse/RTS-982 for 1.4+